pxtlib/docsrender.ts

```diff
@@ -286,7 +286,7 @@ namespace pxt.docs {
         if (theme.homeUrl)
             params["homeurl"] = html2Quote(theme.homeUrl);
         params["targetid"] = theme.id || "???";
-        params["targetname"] = theme.name || "Microsoft MakeCode";
+        params["targetname"] = theme.docsHeader || theme.name || "Microsoft MakeCode";
```

Comment:
let's use another field for the "docs header". "targetname" is used in many other instance and it refers to pxtarget.name.